desktop/src/main/java/bisq/desktop/main/MainView.java

```diff
@@ -413,6 +414,7 @@ protected Tooltip computeValue() {
 
         // Delay a bit to give time for rendering the splash screen
         UserThread.execute(() -> onUiReadyHandler.run());
+        UserThread.execute(() -> checkOpenSupportTickets());
```

Comment:
This should be done more like the new trade protocol window like https://github.com/bisq-network/bisq/blob/95257b10f9741e7d3548c3630dca0b2b2168b366/desktop/src/main/java/bisq/desktop/main/MainViewModel.java#L271 in the MainViewModel. So it is shown after the Splash screen based on priority.

Reply:
Thanks for the suggestions. I will update ASAP.